internal/peer/common/commonpkcs11_test.go

```diff
@@ -0,0 +1,49 @@
+//go:build pkcs11
+// +build pkcs11
+
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package common_test
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hyperledger/fabric/bccsp/factory"
+	"github.com/hyperledger/fabric/internal/peer/common"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBCCSPPKCS11EnvVars(t *testing.T) {
```

Comment:
this test does not occur in the unit test pipelines
<img width="723" alt="image" src="https://user-images.githubusercontent.com/7284783/195323756-c0f0ae56-f4cc-436f-b3e5-9169586f5c42.png">